frontend/src/routes/Infrastructure/Automations/AnsibleAutomationsForm.tsx

```diff
@@ -566,7 +569,6 @@ function EditAnsibleJobModal(props: {
                     label={t('template.modal.name.label')}
                     id="job-name"
                     value={ansibleJob?.name}
-                    helperText={t('template.modal.name.helper.text')}
```

Comment:
The user cannot enter their own name here - if it doesn't match one we fetched from the server, it is cleared. So this text does not make sense.